pom.xml

```diff
@@ -42,6 +42,11 @@
             <groupId>com.h2database</groupId>
             <artifactId>h2</artifactId>
         </dependency>
+        <dependency>
+            <groupId>javax.xml.bind</groupId>
```

Comment:
what is the need for this dependency?

Reply:
I added that accidentally. I will remove it.